src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -9730,6 +9728,8 @@ DECLARE_API(GCWhere)
     return Status;
 }
 
+#ifndef FEATURE_PAL
+
 DECLARE_API(FindRoots)
 {
 #ifndef FEATURE_PAL
```

Comment:
Oops, now my posts are way behind. Your CMakeLists.txt are good. You are good to go. Thanks.